dom/events/EventListener-handleEvent.html

```diff
@@ -81,5 +94,22 @@
 
     target.dispatchEvent(new Event(type));
     assert_true(uncaughtError instanceof TypeError);
-}, "throws if `handleEvent` is not callable");
+}, "throws if `handleEvent` is falsy and not callable");
+
+test(function(t) {
+    var type = "foo";
+    var target = document.getElementById("target");
+    var uncaughtError;
+
+    window.addEventListener("error", function(event) {
+        uncaughtError = event.error;
+    });
```

Comment:
We should maybe add `{ once: true }` to this event listener (and the others in the file) as currently all three of them will run by the time you get to this test. It doesn't matter much as they do the same thing, but it still seems better to account for future changes now and not let someone else try to debug that.

Reply:
Even with `once: true` we risk tests affecting each other in future: non-conformant implementation do not throw `"error"` event.  With current code that would not be a problem, but to make the tests 100% pure I've used `add_cleanup` helper to manually remove `"error"` handlers in a6d0d08.